src/main/java/robot/commands/rumble/MillisecondRumble.java

```diff
@@ -0,0 +1,46 @@
+package robot.commands.rumble;
```

Comment:
get rid of this command, we don't need it since we will just be using the setters in seperate commands